scripts/scil_sh_to_aodf.py

```diff
@@ -55,39 +54,58 @@ def _build_arg_parser():
     shared_group.add_argument('--sigma_spatial', default=1.0, type=float,
                               help='Standard deviation for spatial distance.'
                                    ' [%(default)s]')
 
-    trilateral_group = p.add_argument_group('Angle-aware bilateral arguments')
-    trilateral_group.add_argument('--sigma_angular', default=1.0, type=float,
-                                  help='Standard deviation for angular '
-                                       'distance. [%(default)s]')
-    trilateral_group.add_argument('--sigma_range', default=1.0, type=float,
-                                  help='Standard deviation for range filter.'
-                                       ' [%(default)s]')
+    unified_group = p.add_argument_group('Unified filter arguments')
+    unified_group.add_argument('--sigma_align', default=0.8, type=float,
+                               help='Standard deviation for alignment '
+                                    'filter. [%(default)s]')
+    unified_group.add_argument('--sigma_range', default=0.2, type=float,
+                               help='Standard deviation for range filter\n'
+                                    '*relative to SF range of image*. '
+                                    '[%(default)s]')
+    unified_group.add_argument('--sigma_angle', type=float,
+                               help='Standard deviation for angular filter '
+                                    '(disabled by default).')
+    unified_group.add_argument('--disable_spatial', action='store_true',
+                               help='Disable spatial filtering.')
+    unified_group.add_argument('--disable_align', action='store_true',
+                               help='Disable alignment filtering.')
+    unified_group.add_argument('--disable_range', action='store_true',
+                               help='Disable range filtering.')
+    unified_group.add_argument('--include_center', action='store_true',
+                               help='Include center voxel in neighourhood.')
+    unified_group.add_argument('--win_hwidth', type=int,
+                               help='Filtering window half-width. Defaults to '
+                                    '3*sigma_spatial.')
 
     cosine_group = p.add_argument_group('Cosine filter arguments')
     cosine_group.add_argument('--sharpness', default=1.0, type=float,
-                              help='Specify sharpness factor to use for'
-                                   ' weighted average. [%(default)s]')
+                              help='Specify sharpness factor to use for\n'
+                                   'weighted average. [%(default)s]')
 
-    p.add_argument('--use_gpu', action='store_true',
-                   help='Use GPU for computation.')
+    p.add_argument('--device', choices=['cpu', 'gpu'], default='cpu',
+                   help='Device to use for execution. [%(default)s]')
```

Comment:
Again another option could be to have --use_gpu, which by default would activate --use_opencl.

Again, no opinion.